docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -22,33 +22,35 @@ $$
 h(t) \sim \text{HospDist}\left(H(t)\right)
 $$
 
-Where $h(t)$ is the observed number of hospital admissions at time $t$, and $H(t)$ is the number of latent hospital admissions at time $t$. The distribution $\text{HospDist}$ is discrete. For this example, we will use a negative binomial distribution:
+Where $h(t)$ is the observed number of hospital admissions at time $t$, and $H(t)$ is the number of latent hospital admissions at time $t$. The distribution $\text{HospDist}$ is discrete. For this example, we will use a negative binomial distribution with an inferred concentration.
 
 $$
 \begin{align*}
-h(t) & \sim \text{NegativeBinomial}\left(\text{concentration} = 1, \text{mean} = H(t)\right) \\
-H(t) & = \omega(t) p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau)
+h(t) & \sim \mathrm{NegativeBinomial}\left(\mathrm{mean} = H(t), \mathrm{concentration} = k\right) \\
+H(t) & = p_\mathrm{hosp}(t) \sum_{\tau = 0}^{T_d} d(\tau) I(t-\tau) \\
+\log[p_\mathrm{hosp}(t)] & \sim \mathrm{Normal}(\mu=\log(0.05), \sigma=\log(1.1) \\
+\log(k) & \sim \mathrm{Normal}(\mu=\log(1), \sigma=\log(10))
 \end{align*}
 $$
 
-Were $d(\tau)$ is the infection to hospitalization interval, $I(t)$ is the number of latent infections at time $t$, $p_\mathrm{hosp}(t)$ is the infection to hospitalization rate, and $\omega(t)$ is the day-of-the-week effect at time $t$; the last section provides an example building such a `RandomVariable`.
+Were $d(\tau)$ is the infection to hospitalization interval, $I(t)$ is the number of latent infections at time $t$, $p_\mathrm{hosp}(t)$ is the infection to hospitalization rate.
 
 The number of latent hospital admissions at time $t$ is a function of the number of latent infections at time $t$ and the infection to hospitalization rate. The latent infections are modeled as a renewal process:
 
 $$
 \begin{align*}
-I(t) &= R(t) \times \sum_{\tau < t} I(\tau) g(t - \tau) \\
-I(0) &\sim \text{LogNormal}(\mu = \log(80/0.05), \sigma = 1.5)
+I(t) &= \mathcal{R}(t) \times \sum_{\tau < t} I(\tau) g(t - \tau) \\
+\log[I(0)] &\sim \text{Normal}(\mu=\log(100), \sigma=\log(1.75))
 \end{align*}
 $$
 
-The reproductive number $R(t)$ is modeled as a random walk process:
+The reproduction number $\mathcal{R}(t)$ is modeled as a random walk in logarithmic space, i.e.:
 
 $$
 \begin{align*}
-R(t) & = R(t-1) + \epsilon\\
-\log{\epsilon} & \sim \text{Normal}(\mu=0, \sigma=0.1) \\
-R(0) &\sim \text{TruncatedNormal}(\text{loc}=1.2, \text{scale}=0.2, \text{min}=0)
+\log[\mathcal{R}(t)] & = \log[\mathcal{R}(t-1)] + \epsilon\\
+\epsilon & \sim \text{Normal}(\mu=0, \sigma=0.025) \\
+\mathcal{R}(0) &\sim \text{TruncatedNormal}(\text{loc}=1.2, \text{scale}=0.2, \text{min}=0)
```

Comment:
Not new in this PR, but I am curious about the choice of `TruncatedNormal`.

Reply:
Will be replaced with a Normal on $\log(\mathcal{R}(t=0))$ in #275